packages/utils/src/signatureUtils.ts

```diff
@@ -25,6 +27,34 @@ export interface Signer {
   sign: (message: string) => Promise<string>
 }
 
+export async function addressToPublicKey(
+  signer: string,
+  signFn: (message: string, signer: string) => Promise<string>
+) {
+  const msg = new Buffer('dummy_msg_data')
+  const data = '0x' + msg.toString('hex')
+  // Note: Eth.sign typing displays incorrect parameter order
+  const sig = await signFn(data, signer)
+
+  const rawsig = ethjsutil.fromRpcSig(sig)
```

Comment:
Optional Nit: Didn't know about `fromRpcSig`!! Could we streamline either that or our implementation (from below =`parseSignature`) here?